python/ray/serve/_private/application_state.py

```diff
@@ -186,6 +263,13 @@ def get_app_status(self, name: str) -> ApplicationStatusInfo:
             )
         return self._application_states[name].get_application_status_info()
 
+    def list_app_status(self) -> Dict[str, ApplicationStatusInfo]:
```

Comment:
```suggestion
    def get_app_statuses(self) -> Dict[str, ApplicationStatusInfo]:
```

Reply:
I feel `list` might be accurate, when i use `get`, i either use `get_all_xx ()` or `get(name)`.

Reply:
I'm okay with `list` or `get`, but can we change `app_status` to `app_statuses`? The function returns multiple app statuses, so it's more accurate to use the plural.

Reply:
good point